tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/object/ObjectInBalaTest.java

```diff
@@ -546,6 +546,11 @@ public void testObjectReferingTypeFromBalaNegative() {
                 "no implementation found for the method 'toString' of class 'FrameImpl'", 53, 1);
     }
 
+    @Test(description = "Test invoking remote function in client object with inclusion")
```

Comment:
Although we are using the invocation to test, we are not testing the invocation? We are testing inclusion and param names. I would use a name/description that describes what we are actually testing.

Reply:
Ah yeah, Thats correct. Will change